tests/Queue/RedisQueueIntegrationTest.php

```diff
@@ -303,6 +305,7 @@ public function testSize()
         $this->assertEquals(3, $this->queue->size());
         $job = $this->queue->pop();
         $this->assertEquals(3, $this->queue->size());
+        $job->getContainer()->shouldReceive('make')->once()->with('Illuminate\Queue\CallQueuedHandler')->andReturn($handler = m::mock('StdClass'));
```

Comment:
If I do that I'm getting

> No matching handler found for Mockery_27_Illuminate_Container_Container::make("Illuminate\Queue\CallQueuedHandler"). Either the method was unexpected or its arguments matched no expected argument list for this method

Which is weird since it's exaclty the same string ...